### ISSUE TITLE
Running Command php artisan seat:admin:reset throws exeption

### ISSUE THREAD

**Issue by mthomas92868** [eveseat/seat#150]
V2.0.0-Alpha2  

```
[Symfony\Component\Debug\Exception\FatalThrowableError]
  Type error: Return value of Seat\Console\Commands\Seat\Admin\Reset::getCompleteRole() must be an instance of Illuminate\Support\Collection, instance of Seat\Web\Mo
  dels\Acl\Role returned
```

**Comment by leonjza** [comment]
Thanks. This will be fixed before the final 2.0 release.

**Comment by leonjza** [comment]
Actually, this is already fixed in https://github.com/eveseat/web/commit/fa2a4fd9b695fba6a59592456c90e250dd36dd28 and will be available in next tagged release.

**Comment by mthomas92868** [comment]
Thanks I manual made the changes, and it worked fine this time....

**Comment by mthomas92868** [comment]
Fixed in 2.0.0
